### DISCUSSION TITLE
Stable Payments with PYUSD

### DISCUSSION THREAD

**Discussion by rubgithub** [akash-network/community#296] [Ideas]
PayPal:
_According to PayPal (PYUSD), it is a stablecoin backed by secure and highly liquid assets. As PYUSD is introduced, you will have the ability to purchase, sell, retain, and transfer it through our application or on our website._

https://www.paypal.com/us/digital-wallet/manage-money/crypto/pyusd

**Comment by bmenzalji** [comment]
If there is demand from the community for PYUSD for a payment option on Akash, we're happy to explore voting for it.  There might be willing providers to want to accept PYUSD and be in the Paypal ecosystem for support, but it's too early to tell.

PYUSD, in my opinion, is more geared for ecommerce solutions and is a better market fit there than cloud payment rails.

**Reply by Lucas-Kohorst** [reply]
ya agree... is pyusd even available in the cosmos ecosystem?

**Reply by Lucas-Kohorst** [reply]
which also brings up another point curious how you guys are thinking about https://nobleassets.xyz/

**Comment by bmenzalji** [comment]
PYUSD isnt in the Cosmos ecosystem, not that we are aware of.

As for Noble Assets, I've already reached out to Jelena when they announced native USDC last year.  We'll be patient and wait for implementation to see it, and then make a governance proposal once we have confirmation there aren't any technical hiccups that prevent us from implementing on Akash.  

Great questions though.

**Comment by rodri-r** [comment]
I will never use PYUSD :)

One of the main goals of crypto is to not have intermediaries and pay their ridiculous fees. 

PayPal used to ban/block users for having accounts related to Bitcoin, Litecoin and other crypto assets.  As well as for many other reasons.

Now that they see that crypto is here to stay and that they are being left behind, they want a piece of the pie too.  They will most likely still ban/block users for whatever reason they want. PayPal and its new PYUSD is worse than most CEXs.

https://cointelegraph.com/news/paypal-suspends-user-for-crypto-trading-using-paypal-s-own-service

**Reply by piber-dev** [reply]
not using pyusd out of principle is understandable, however i am of the opinion that on- & off-ramps are great. the more and the easier integrations are built, the better for the adoption of crypto. if some of these integrations were to rug-pull or fail, that's also not surprising, but the service is worth a lot when it works. if pyusd would open up for companies that perhaps only uses paypal, that would mean that they could use akash.

**Reply by nick134-bit** [reply]
imo there will be many more stable coins in the future and akash cant whitelist all of them, it should just be easy for users to access the stable token akash supports through (IBC) swaps.

**Reply by piber-dev** [reply]
i like this take @nick134-bit, seeing how we already have stable coins guess it's not much of a priority. perhaps the priority for other projects is to enable swapping between these if it is not already available, and not for akash to waste time on yet another integration.

**Reply by pcfreak30** [reply]
I believe that handling payments via a DEX over IBC would be better than taking direct payment?. Only concern might be tax implications with the state. The USA is still anti-crypto for the next 5 years minimum.
